### ISSUE TITLE
Prepare alpha release of network groups

### ISSUE THREAD

**Issue by hsablonniere** [CleverCloud/clever-tools#566]
* uncomment NG commands
* add a "alpha/feedback" banner on all NG commands
* prepare short URL https://go.clever-cloud.com/feedback

```
Logger.println(colors.yellow(`️😉 Hey, you found an alpha feature, please give us your feedback at https://go.clever-cloud.com/feedback`));
```

**Comment by KannarFr** [comment]
Do you have a tool idea to get feedback?

**Comment by hsablonniere** [comment]
> Do you have a tool idea to get feedback?
No, Steven is supposed to find/chose one. We also need a short URL system to change the target if necessary.

**Comment by KannarFr** [comment]
@StevenLeRoux, any bumps?

**Comment by StevenLeRoux** [comment]
working on it

**Comment by KannarFr** [comment]
What about a link go.clever-cloud.com/networkgroups that forward to a custom discord channel on our discord?

**Comment by hsablonniere** [comment]
Decision:

* start with direct link to `https://github.com/CleverCloud/Roadmap/discussions/new?category=alpha&title=[NetworkGroups]`
  * pass the "title" as a param of `displayAlphaBanner` as a const
* next time, we could use a URL shortner
